src/integration_tests/deploy_prism/definition.ts

```diff
@@ -259,17 +270,14 @@ async function prism_init_verbose(
 		yluna_token_addr,
 		prism_governance_info.xprism_token_addr,
 		prism_governance_info.prism_gov_deployment_addr,
+		prism_xprism_boost_addr,
 	);
 	console.log(`prism_launch_pool instantiated\n\taddress: ${prism_launch_pool_addr}`);
 	console.log(`=======================`);
 
-	// instanstiate boost contract
-	let prism_xprism_boost_addr = await init_prism_xprism_boost(
-		lcd_client,
-		sender,
-		prism_governance_info.xprism_token_addr
-	)
-	console.log(`prism_xprism_boost instantiated\n\taddress: ${prism_xprism_boost_addr}`);
+	const msg_update_config = { update_config: { launch_pool_contract: prism_launch_pool_addr } };
+	await execute_contract(lcd_client, sender, prism_xprism_boost_addr, msg_update_config);
```

Comment:
@HoldMyBeer1111 can we move this contract call to inside `init_prism_xprism_boost` function if it needs this msg to launch? just to keep it together into 1 function

Reply:
guess no, we have to instantiate launch_pool at first, which already requires xprism_boost address

Reply:
gotcha sg, the pr looks good